hmda/src/main/scala/hmda/api/http/admin/InstitutionAdminHttpApi.scala

```diff
@@ -49,34 +49,14 @@ trait InstitutionAdminHttpApi extends HmdaTimeDirectives {
         val institutionPersistence = sharding.entityRefFor(
           InstitutionPersistence.typeKey,
           s"${InstitutionPersistence.name}-${institution.LEI}")
-
-        timedPost { uri =>
-          val fCreated: Future[InstitutionCreated] = institutionPersistence ? (
-              ref => CreateInstitution(institution, ref))
-
-          onComplete(fCreated) {
-            case Success(InstitutionCreated(i)) =>
-              complete(ToResponseMarshallable(StatusCodes.Created -> i))
-            case Failure(error) =>
-              val errorResponse =
-                ErrorResponse(500, error.getLocalizedMessage, uri.path)
-              complete(ToResponseMarshallable(
-                StatusCodes.InternalServerError -> errorResponse))
-          }
-        } ~
-          timedPut { uri =>
-            val fModified: Future[InstitutionEvent] = institutionPersistence ? (
-                ref => ModifyInstitution(institution, ref)
-            )
-
-            onComplete(fModified) {
-              case Success(InstitutionModified(i)) =>
-                complete(ToResponseMarshallable(StatusCodes.Accepted -> i))
-              case Success(InstitutionNotExists(lei)) =>
-                complete(ToResponseMarshallable(StatusCodes.NotFound -> lei))
-              case Success(_) =>
-                complete(
-                  ToResponseMarshallable(HttpResponse(StatusCodes.BadRequest)))
+        path("institutions") {
```

Comment:
This might need to be placed higher up.  Any requests to `/gibberish` will first try and parse the request body as an Institution, before giving an error like `The request content was malformed`.  Only if you've given a properly formatted Institution will the application respond with the appropriate `404: The requested resource could not be found.`